content/blog/wingingit/winging-it-06.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Proxy State
+episode: 6
+tags:
+  - JavaScript
+  - Proxy
+  - State Management
+author:
+  - james
+  - sondra
+  - stacy
+date: 2024-04-18
+length: 27 mins
+image:
+  src: winging-it/winging-it-6.jpg
+media:
+  - youtube: gs94cnKXHh4
+summary: |
+  For simple interactive sites, using a proxy to manage your state and side
+  effects may be enough.
+---
+
+{% import "embed.macros.njk" as embed %}
+{% import "utility.macros.njk" as utility %}
+
+{{ embed.figure(
+  data=media
+) }}
+
+{{ utility.main_action(
+  'Subscribe to Channel »',
+  subscribe_url
+) }}
+
+## What we cover:
+
+- What is state?
+- State management solutions
+- What are Proxies?
+- When and How to use Proxies
+- When to avoid using Proxies
+- Proxies in the real world
+
+## Links:
+
+- [Proxy on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy)
+- [Signals Proposal](https://github.com/proposal-signals/proposal-signals)
+- [Proxy is What’s in Store](https://www.oddbird.net/2024/01/12/proxy-store/)
+- [Sass](https://sass-lang.com/)
```

Comment:
```suggestion
- [Sass Playground](https://sass-lang.com/playground)
```

Reply:
@jamesnw I added [this link](https://sass-lang.com/playground/#eJxdj9FqwjAUhu/zFD9OVgULyhAhvfFV0ubYhiY5JYk4J333xdluZTch5zv5v58cDus6kOoHNj5FiY1AEZ2ythxUS4XExz6Q22XqSJurm/HxOGGrQkszPb3othJCnEk1HRb2HdaxCUS+jOaLYPxyGfEQwPlZovB6BuU1Ns748mZ06iSKt8fSMBbbnxBQs75PV+BdypouHOiXAA37RD5JrLLir3TEwv5Pvqqm9CjmcxRj/tezbFlRq6ZvA1+9Lhu2HCQSO5U456f51plEedQmDlbdJWrLTZ/BwNEkw17iYj5JZ5J4kNhXYvwGtNZ+7g==) to some sample code already in place